python_bot.py

```diff
@@ -19,9 +19,11 @@
 from threading import Timer
 import time
 import urllib.request
+import sys
 
 # local file imports
 import config
+import commands.help as help
```

Comment:
very nice, I like this a lot more. Can the last word be anything? for example is the below valid...
`import commands.help as i_do_what_i_want`
then use it like
`i_do_what_i_want.command(client, message)`

Just currious

Reply:
exactly